README.md

```diff
@@ -20,6 +20,21 @@ Users can [**subscribe** to **notifications**](https://github.com/ambanum/CGUs-v
 
 _Note: For now, when multiple versions coexist, **terms** are only **tracked** in their English version and for the European jurisdiction._
 
+## Description de l'outil
+
+_Note: Les mots en gras sont les [termes du domaine](https://en.wikipedia.org/wiki/Domain-driven_design)._
```

Comment:
Link to the french web page `https://fr.wikipedia.org/wiki/Conception_pilot%C3%A9e_par_le_domaine` :)